alerts/UA00002.json

```diff
@@ -1,9 +1,11 @@
 {
   "dateRaised": 20201028,
-  "affectedFirmware": ["all"],
+  "affectedMasterFirmware": ["all"],
```

Comment:
By the way, GitHub is moving away from the word "master" for the main branch.  I forget what it will be called (maybe "main"?) but we also use the word "latest".